js/src/components/paid-ads/constants.js

```diff
@@ -0,0 +1,282 @@
+/**
```

Comment:
💅  I would name this `specs.js` or `specs-config.js` to be less generic

Reply:
Renamed to `asset-specs.js` by https://github.com/woocommerce/google-listings-and-ads/pull/1863/commits/856763cd1103f707523f1399a526b7da7baf5ce6.